### PR TITLE
Create blog post on :is() :where() :has() CSS pseudo selectors

### PR DESCRIPTION
**Pull request by bigandy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
     "generate-post": "node generatePost.js"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.15",
-    "@docusaurus/preset-classic": "2.0.0-beta.15",
+    "@docusaurus/core": "2.0.0-beta.18",
+    "@docusaurus/preset-classic": "2.0.0-beta.18",
```
Comment (royletron) [inline package.json]: booo, also did the update work? I always get so many errors when updating hence why I locked it to a specific beta?

Comment (royletron) [reply package.json]: ya, I just pulled and tried run dev and it's very unhappy

File: src/theme/components/blog-layout/blog-layout.scss [hunk 2]
```diff
@@ -6,15 +6,21 @@ div[class^="authorCol_"] {
...
-  .blog-wrapper {
+  .main-wrapper {
     .container {
       padding: 0 !important;
       margin: 0;
       max-width: initial;
-      .markdown {
-        margin-bottom: 30px;
-      }
+
```
Comment (royletron) [inline src/theme/components/blog-layout/blog-layout.scss]: This seems to mess with stuff on the careers pages?

**Review by royletron — CHANGES_REQUESTED** [review]
generally I would separate your post stuff from the build....

File: package.json (hunk 1 above)
Comment (bigandy) [reply package.json]: works fine for me, if I use node version 14.16.1. I can seperate the post from the build into two PRs.

File: src/theme/components/blog-layout/blog-layout.scss (hunk 2 above)
Comment (bigandy) [reply src/theme/components/blog-layout/blog-layout.scss]: have separated into new PR for this code.

Comment (royletron) [reply src/theme/components/blog-layout/blog-layout.scss]: @bigandy looks like it's still here in this one?

Comment (bigandy) [reply src/theme/components/blog-layout/blog-layout.scss]: noooo. Sorry. Too many branches going on at the moment!

Comment (bigandy) [reply src/theme/components/blog-layout/blog-layout.scss]: done. and also used :where() in the css

**Pull request merged**